code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -105,7 +105,7 @@
 	if(isliving(user))
 		var/mob/living/L = user
 		if(HAS_TRAIT(L, TRAIT_SKITTISH))
-			. += "<span class='notice'>Ctrl-Shift-click [src] to jump inside.</span>"
+			. += "<span class='notice'>If you bump into [src] while running, you will jump inside.</span>"
```

Comment:
i think simply "it" instead of [src] would sound better here?

Reply:
`[src.p_them()]`